main.cpp

```diff
@@ -13,75 +13,486 @@ What you create in this project will be used as the basis of Project 5 in the co
+    //provide classes
+    void provideClasses ();
+ };
+
+/*
+Thing 2) Games Console
+5 properties: 
+    1) Amount of storage memory (float)
+    2) Amount of downloaded games (int)
+    3) Amount of power consumed per hour rms (float)
+    4) HDMI Input and output (y/n) (bool)
+    5) Amount of usb slots (int)
+3 things it can do:
+    1)Connect to the internet
+    2)Play games from a disk
+    3)Play Videos
+ */
+struct GamesConsole
+{
+    //Amount of storage memory 
+    float storageInSSD = 256.f;
+    //Amount of downloaded games 
+    int downloadedGames = 4;
+    //Amount of power consumed per hour rms 
+    float WattsPerHourRMS = 100;
+    //HDMI Input and output 
+    bool hasHDMI= true; 
+    //Amount of usb slots 
+    int usbSlots = 4;
+    //Connect to the internet
+    struct HarrysConsole
+    {
+       //digital games on harrys console
+       int digitalGames = 10;
+       //number of controller
+       int controller = 2;
+       //Brand-new yes
+       bool isBrandNew = false;
+       //brand
+       std::string brand = "Playstation";
+       //price
+       float price = 499.99f;
+
+       void swapConsoleWithHarry();
+       void TurnOnHarrysConsole ();
+       float askHarryPrice ();
+    };
+
+    void connectToInternet ();
+    //Play games from a disk
+    void playGameFromDisk ();
+    //Play Videos
+    void playVideo ();
+};
+/*
+Thing 3) Rollercoaster
+5 properties:
+    1) Amount of seats (int)
+    2) number of max speed mph (int)
+    3) Average number of laps a day (int)
+    4) Amount of money to ride (float)
+    5) Peak Number of riders a day (int)
+3 things it can do:
+    1) stop
+    2) slow down
+    3) speed up
+ */
+struct Rollercoaster
+{
+    //Amount of seats (int)
+    int amountOfSeats = 40;
+    //number of max speed mph (int)
+    int maxSpeedMph = 80;
+    //Average number of laps a day (int)
+    int averageLaps = 30;
+    //Amount of money to ride (float)
+    float cost = 2.99f;
+    //Peak Number of riders a day (int)
+    int ridersPerDayInPeakTime = 200;
+    //stop
+    void stop ();
+    //slow down
+    void slowDown ();
+    //speed up
+    void speedUp ();
+};
+/*
+Thing 4) Art Musuem
+5 properties:
+    1) Number of works of art (int)
+    2) Number of statues (int)
+    3) Most iconic piece (std::string" "")
+    4) Number of visitors (int)
+    5) Number of new pieces (int)
+3 things it can do:
+    1)Add new pieces
+    2)Remove old pieces
+    3)Restore ancient art */
+
+struct artMusuem
+{
+//Number of works of art (int)
+int worksOfArt = 90;
+//Number of statues (int)
+int statues = 20;
+// Most iconic piece (std::string" "")
+std::string mostFamous = "TheMonaLisa";
+//) Number of visitors (int)
+int vistorsPerYear = 19000;
+//Number of new pieces (int)
+int NumberOfNewPieces = 20;
+//Add new pieces
+void addNewArt (bool statue = true, bool painting = false);
+//Remove old pieces
+void removeOld ();
+//Restore ancient art
+void restoreArt ();
+};
```

Comment:
- Fix this indentation
- Type names need to start with an Upper Case letter
- variable names need to start with a lower-case letter
- [ ] revise